content/en/checkout/reference.md

```diff
@@ -11,7 +11,7 @@ category: Checkout
 
 `POST`
 
-Creates a Checkout Session.
+Creates a Checkout Session. [API URLs](/api/overview#urls)
```

Comment:
Maybe add a paragraph right below the API title, instead of adding this link here. I'm thinking later when we add more service calls we don't want to add the URLs link to each one.